spec/spec_helper.rb

```diff
@@ -99,5 +99,7 @@
   #   Kernel.srand config.seed
 end
 
+# WebMock.enable_net_connect!
```

Comment:
Question: Is this meant to be left here? If so, maybe a comment to say why?

Reply:
Comment added.